test/ess-test-indentation.el

```diff
@@ -86,7 +88,9 @@ where the edit took place. Return nil if E represents no real change.
   (with-current-buffer buffer
     (setq buffer-undo-list nil)
     (indent-region (point-min) (point-max))
-    (not (buffer-modified-p buffer))))
+    (if (buffer-modified-p buffer)
+        (progn (switch-to-buffer buffer) nil)
+      t)))
```

Comment:
Why this change? The previous way seems more straightforward.

Reply:
Just convenience for the interactive use. In case of error the buffer with the wrong indentation will be brought to the front.